### DISCUSSION TITLE
Parallax camera movement

### DISCUSSION THREAD

**Discussion by Cjjuntti** [keiwando/toonsquid#340] [Ideas]
Hey, so I’ve been using toonsquid for about 4 months now, and it’s probably the BEST app I’ve ever used. The camera and keyframes are nice, however my problem with the camera is that all the layers move with it at the same time, which looks unnatural to me. I really want an option for a parallax camera, which makes things in the background move slower, and makes things in the foreground move faster.

**Comment by DWrightAnimation** [comment]
I know what you mean, I wish that could come, too. However, I was able to make the parallax effect manually on ToonSquid, I just had to move each layer according to perspective. I hope this inspires you to look for solutions and workarounds for other problems you encounter. https://github.com/keiwando/toonsquid/assets/129681998/919f1cd9-6976-41cd-bdae-e7d85ee64044
